From a958cf2966293982433dc628bbaf5a4ca46bd665 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:12:25 -0400
Subject: [alsa] cs5535audio: use named constants for pci_power_t values

Message-id: <140972823573-13-patchset-jkysela@redhat.com>
Patchwork-id: 90713
O-Subject: [RHEL 7.1 PATCH 012/442] ALSA: cs5535audio: use named constants for pci_power_t values
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The semantic patch that makes this change is as follows:
(http://coccinelle.lip6.fr/)

// <smpl>
@@
expression pdev;
@@

pci_set_power_state(pdev,
- 3
+ PCI_D3hot
)
// </smpl>

Signed-off-by: Julia Lawall <Julia.Lawall@lip6.fr>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: ec06b2bea73ed0db1e400ff5143c14e31836a77c)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/cs5535audio/cs5535audio.c b/sound/pci/cs5535audio/cs5535audio.c
index 902bebd..c0d2835 100644
--- a/sound/pci/cs5535audio/cs5535audio.c
+++ b/sound/pci/cs5535audio/cs5535audio.c
@@ -253,7 +253,7 @@ static irqreturn_t snd_cs5535audio_interrupt(int irq, void *dev_id)
 static int snd_cs5535audio_free(struct cs5535audio *cs5535au)
 {
  synchronize_irq(cs5535au->irq);
- pci_set_power_state(cs5535au->pci, 3);
+ pci_set_power_state(cs5535au->pci, PCI_D3hot);
 
  if (cs5535au->irq >= 0)
   free_irq(cs5535au->irq, cs5535au);
-- 
1.7.1